### DISCUSSION TITLE
Unable to create workspaces

### DISCUSSION THREAD

**Discussion by ppatierno** [kcp-dev/kcp#1285] [General]
I have build and started kcp as described in the landing README page and was taking a look at the last demo video.
First off all if I understood correctly (as the video mentions) I should have an Organization workspace created by default but I cannot see it.

```shell
kubectl get workspaces
No resources found
```

It seems I am on the "root:default" workspace anyway by running

```shell
kubectl kcp ws
Current workspace is "root:default".
```

But when try to create a new workspace as in the demo I got the following error.

```shell
kubectl kcp ws create --type Organization green
error: clusterworkspaces.tenancy.kcp.dev "green" is forbidden: spec.type "Organization" in workspace "root:default" does not exist
```

Any idea what's going on? I am just trying to follow the guide and video steps.

**Comment by ncdc** [comment]
As of today, you start out in `root:default` which is already an Organization workspace. Inside there, you can create Universal workspaces. If you want to create additional Organization workspaces, you need to first do `kubectl ws root`.

We will be moving to a more flexible model, but we aren't fully there yet.

**Reply by ppatierno** [reply]
Thanks for the immediate answer!!
Yeah you are right, I had to move to "root" workspace.
I see the project evolving fast, so demos, videos and presentations are not really up to date anymore. 
Any chance to have a more detailed tutorial/getting-started to be up to date? (for now I will continue to try ;-))
I think it needs to lower the barrier for new users starting up with kcp.

**Reply by ncdc** [reply]
We hear you 😄. We're doing our best, but given the pace at which the project is advancing, it's hard to keep all the demos and docs up to date. We'll see if we can get some basic ones added/updated, though.

**Reply by ppatierno** [reply]
It's totally understandable, it's just that it could put on you more work about answering stupid questions (like mines) for setting up everything :-)

**Reply by aishabbah** [reply]
Hello can you tell me how you were able to get the kcp commands to work.
When I type :  k kcp ws create --type Organixation green
I get 
error: unknown command "kcp" for "kubectl"

Did you mean this?
	top
	cp
aisha@Aishas-MacBook-Pro kcp % 
Do you know what I could be doing wrong ?

**Reply by kasturinarra** [reply]
Not sure if you have already done below, if not you will need to do the same to be able to get kubectl kcp working for you, either you have not installed kubectl kcp plugin or it is not in your path, either of these would be the cause for your problem. Hope that helps !!
(requires go 1.17 or later)
$ make install WHAT=./cmd/kubectl-kcp
$ make install WHAT=./cmd/kubectl-workspaces
$ make install WHAT=./cmd/kubectl-ws
If $HOME/go/bin is not in your $PATH, you’ll need to add it, or feel free to move $HOME/go/bin/kubectl-kcp to someplace that is in your $PATH

**Reply by aishabbah** [reply]
Thanks that work, it was a mixture of both, my path needed to be altered and also I needed to install those three plugins.
